### PR TITLE
Update ap vendor packages

### PR DESCRIPTION
**Pull request by himabindu07**
## Details
Update ap vendor packages aws-es-proxy, git-daemo, pgbouncer_exporter, Postgres-exporter, registry.

Do not merge this PR until this description is replaced with relevant info about what images are being updated and what changes are being made.

## Related Issues

Do not merge this PR until this text is replaced with links to related issues if one exists, or else this section is deleted.

## Checklist

- [ ] version.txt was updated

### CODE REVIEW

**Comment by pgvishnuram** [comment]
Pls add relavent issue ticket for the same to track for future purpose and also a note on what this PR is addressing

File: registry/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ COPY parser.go $DISTRIBUTION_DIR/configuration/parser.go
 # This step is required to generate build config
 FROM base AS version
 ARG PKG="github.com/docker/distribution"
-ENV REGISTRY_VERSION 2.8.2
+ENV REGISTRY_VERSION 2.8.3
```
Comment (pgvishnuram) [inline registry/Dockerfile:31]: registry service is still broken you can release image in ap-vendor but lets not point in astronomer/astronomer

**Review by danielhoherd — APPROVED** [review]

File: pgbouncer-exporter/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-0.15.0-2
+0.16.0
```
Comment (pgvishnuram) [inline pgbouncer-exporter/version.txt:1]: why pgbouncer version has been updated it should be 0.15.0-3

Comment (danielhoherd) [reply pgbouncer-exporter/version.txt:1]: good catch!

Comment (danielhoherd) [reply pgbouncer-exporter/version.txt:1]: @pgvishnuram i refreshed your PR with changes that will correct this mistake https://github.com/astronomer/ap-vendor/pull/619

**Pull request merged**